### DISCUSSION TITLE
[Script request] BirdNET

### DISCUSSION THREAD

**Discussion by Depechie** [community-scripts/ProxmoxVE#1078] [Request script] [answered]
### Application Name

BirdNET

### Website

https://github.com/tphakala/birdnet-go

### Description

BirdNET can be used to monitor and record bird sounds and identify the actual bird that produces the sound.
There are several different container versions online to select from, I personally had good experience with the BirdNET-go one that I linked.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by uSlackr** [comment]
I can confirm no duplicates, nothing existing and it is not game rleated.

**Reply by troymroberts** [reply]
Also came here looking to see if there was a request

**Comment by MickLesk** [comment]
The Repo pushes since nearly 1 year pre releases. Anyone know why? Because we dont use prereleases.

**Comment by MickLesk** [comment, marked as answer]
https://github.com/community-scripts/ProxmoxVED/issues/1615
